quickwit/quickwit-indexing/src/split_store/indexing_split_store.rs

```diff
@@ -653,18 +741,21 @@ mod test_split_store {
         fs::write(root_path.join("b.split").join("termdict"), b"b").await?;
         fs::write(root_path.join("c.split").join("termdict"), b"c").await?;
 
-        let cache_params = IndexingSplitStoreParams {
-            max_num_splits: 100,
-            max_num_bytes: 200,
-        };
+        let cache_size_checker = Arc::new(Mutex::new(SplitStoreSpaceQuota::new(100, 200)));
         let remote_storage = Arc::new(RamStorage::default());
         let merge_policy = Arc::new(StableMultitenantWithTimestampMergePolicy::default());
         let split_store = IndexingSplitStore::create_with_local_store(
             remote_storage,
             local_dir.path(),
-            cache_params,
             merge_policy,
-        )?;
+            cache_size_checker.clone(),
+        )
+        .await?;
+        let initial_size = {
+            let cache_size_checker_lock = cache_size_checker.lock().await;
```

Comment:
We tend to distinguish guards from locks in the rest of the codebase so let's use `*_guard` for guards and `_lock` for locks in this PR.